CMakeLists.txt

```diff
@@ -326,29 +326,50 @@ IF(NOT CYCLUS_DOC_ONLY)
     ####################################### begin cpack ##########################################
     ##############################################################################################
 
-    SET(CPACK_COMPONENTS_ALL cyclus testing libraries data core)
 
     # Here's where we package it with CPack
     SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Cyclus : A nuclear fuel cycle simulator from UW-Madison.")
-
+    
+    # Here we set some components for installation with cpack
+    SET(CPACK_COMPONENTS_ALL cyclus testing libraries data core)
     SET(CPACK_GENERATOR "DEB")
+
+    # Organisation
     SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "CNERG, UW-Madison") #required
     SET(CPACK_PACKAGE_VENDOR "CNERG, UW-Madison")
     SET(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.rst")
     SET(CPACK_RESOURCE_FILE_LICENSE "${PROJECT_SOURCE_DIR}/LICENSE.rst")
+
+    # Version
     SET(CPACK_PACKAGE_VERSION_MAJOR "1")
-    SET(CPACK_PACKAGE_VERSION_MINOR "0")
-    #SET(CPACK_PACKAGE_VERSION_PATCH "0") # should use commit number here  ?
-    SET(CPACK_PACKAGE_NAME "${CMAKE_PROJECT_NAME}")
+    SET(CPACK_PACKAGE_VERSION_MINOR "4")
+    SET(CPACK_PACKAGE_VERSION_PATCH "0") 
```

Comment:
Is there somewhere to add a modifier, like "RC" for a release candidate, to this version string?  (I'm being a little pedantic here)